### ISSUE TITLE
Shinra Meter Problem! Help! D:

### ISSUE THREAD

**Issue by Sodenia11** [neowutran/ShinraMeter#307]
Hey!

I want ot install shinrameter for TERA via this website: github.com/neowutran/ShinraMeter/tree/master

When I install it and open the file "ShinraMeter-master" I only can find "ShinraLauncher.exe" file but not the actual shinra meter!

What should I do? Please help

Also: When I use later versions it always says when I open the shinrameter.exe ""Unable to contact update server, try again later, additional data in error.log" and when I am ingame it says "A fatal error has been found, additonal data in error.log"

. I don't know why.. I always used SM and everthing was alright but since yesterday it won't work anymore :( I tried everything to fix this...

**Comment by neowutran** [comment]
Holla 
github.com/neowutran/ShinraMeter/tree/master only contains the code source files, not the directly usable software. 

Shinra meter server is offline atm due to technical failure on it

You can download the shinra from here as a temp solution
http://rgho.st/8tkP9ljXx

& if you see "A fatal error has been found, additonal data in error.log", send the content of error.log file

**Comment by Sodenia11** [comment]
Hey,

Thanks a lot for the newest version. It actually woooooooooooooooooorked! :-))))

Since I did keep getting this messages with the older versions I tried - 1.71 and 1.76, here's what the last one had if it would be useful. A friend said he had no trouble using 1.76 whatsoever. He's with Win 7 if it matters at all. I'm with 10.

[error.txt](https://github.com/neowutran/ShinraMeter/files/851395/error.txt)

Keep up the good work, dude!

Cheers!
